zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/handlers/FormHandler.java

```diff
@@ -55,7 +55,7 @@ public void updateEntity(final Record<Form> record, final FormEntity entity) {
     entity
         .setVersion((long) value.getVersion())
         .setKey(value.getFormKey())
-        .setBpmnId(value.getResourceName())
+        .setBpmnId(value.getFormId())
```

Comment:
❌ ❓ This seems a bit weird to me. Is this expected? Why is the property called bpmnId but we fill the formId? 🤔 Can we rename the property then?

Reply:
This is how it was back on Tasklist's original Index and Entity. I maintained the same notation across while doing the ports.